src/components/organisms/NewProfileModal/components/header/ProfileModalAvatar/ProfileModalAvatar.tsx

```diff
@@ -79,7 +76,7 @@ export const ProfileModalAvatar: React.FC<ProfileModalAvatarProps> = ({
           imageClassName="ProfileModalAvatar__image"
           user={userWithOverriddenPictureUrl}
           size="full"
-          showStatus={!isCurrentUser}
+          showStatus
```

Comment:
fixed status displaying for edit window based on https://github.com/sparkletown/internal-sparkle-issues/issues/1108#issue-979103157